src/target/target-opening-hours/TargetOpeningHours.scss

```diff
@@ -0,0 +1,78 @@
+@import 'node_modules/hds-design-tokens/lib/all.scss';
+
+.target-opening-hours-header {
+  min-height: $spacing-layout-l;
```

Comment:
I would use some own custom value here: 4 rem or something.
Because of semantics and the value might change, very unlikely but still.

Reply:
- [ ] Apply custom value instead of spacing-layout-l variable